sql/src/main/java/io/crate/analyze/PrivilegesAnalyzer.java

```diff
@@ -66,14 +113,14 @@ private static void ensureUserManagementEnabled(@Nullable User user) {
         }
     }
 
-    private static Collection<Privilege.Type> getPrivilegeTypes(boolean all, List<String> typeNames) {
```

Comment:
pls rebase again and don't remove optimized code but adjust your code instead ;)